aws/resource_aws_acm_certificate_test.go

```diff
@@ -17,16 +17,19 @@ import (
 
 var certificateArnRegex = regexp.MustCompile(`^arn:aws:acm:[^:]+:[^:]+:certificate/.+$`)
 
-func TestAccAwsAcmCertificate_emailValidation(t *testing.T) {
+func testAccAwsAcmCertificateDomainFromEnv(t *testing.T) string {
 	if os.Getenv("ACM_CERTIFICATE_ROOT_DOMAIN") == "" {
 		t.Skip("Environment variable ACM_CERTIFICATE_ROOT_DOMAIN is not set")
 	}
+	return os.Getenv("ACM_CERTIFICATE_ROOT_DOMAIN")
```

Comment:
Can you add any brief instructions in a comment on what is this domain in case it's not clear from the ENV variable name?

e.g.

```go
// ACM_CERTIFICATE_ROOT_DOMAIN is a domain that was previously registered
// and successfully verified via ACM either via email or DNS
// See https://us-west-2.console.aws.amazon.com/acm/home
```
Also I assume the verification only needs to be done for the TLD (root) domain (e.g. `terraformtesting.com`), not for SAN or anything else?

Reply:
Unfortunately, its more complicated then that. Its per domain in each certificate request done via email validation. So if I request foo.example.com and bar.example.com with email validation, I will get two approval emails for that single request. If I go to renew them, I have to click the emails again. Hence why everyone was clamoring for DNS validation 😁

Reply:
But I'll definitely add information about the environment variable, its non-descriptive at the moment.